src/main/java/org/folio/consortia/service/impl/UserServiceImpl.java

```diff
@@ -97,6 +99,7 @@ public User prepareShadowUser(UUID userId, String tenantId) {
             .firstName(userOptional.getPersonal().getFirstName())
             .lastName(userOptional.getPersonal().getLastName()));
         }
+        user.setCustomFields(Map.of(CUSTOM_FIELD_NAME, tenantId));
```

Comment:
please rename, if I look with code without idea I could not imagine what this custom field about. It should be self descriptive. I propose name: ORIGINAL_TENANT_ID

Reply:
fixed